src/app/find/mapPage/MapPage.jsx

```diff
@@ -306,9 +308,9 @@ export default class MapPage extends Component {
                   <ResultsBar
                     isSearching={this.state.isSearchingLocations}
                     filterString={this.getCurrentFilterString()}
-                    hasResults={!!this.state.locations.length}
+                    hasResults={this.state.hasResults}
```

Comment:
This is actually an integer rather than a boolean, since it's `hasResults: this.state.locations && this.state.locations.length` without `!!`, but that doesn't actually matter.